osu.Game/Screens/Play/HUD/GameplayPerformancePointsCounter.cs

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
```

Comment:
Can you split this into its own PR?

Reply:
@frenzibyte in the mean time, you could still look at this one.

Reply:
PP counter reverted in https://github.com/ppy/osu/pull/25226/commits/58a830fc5b2bd749ebccd7cf1b5e0f12b6f4c117, https://github.com/ppy/osu/pull/25226/commits/a23dfbeab55bb3e195407bcf436444f150d5a355, https://github.com/ppy/osu/pull/25226/commits/6c3169a0ed6e9f0f9e1e7d668993e04861f63540, and https://github.com/ppy/osu/pull/25226/commits/77f5a4cdf53fca660a71140b84c9897439ddbf2e. Now reside in https://github.com/ppy/osu/pull/25362 instead.